google-cloud-bigtable/Rakefile

```diff
@@ -174,6 +174,34 @@ task :doctest do
   sh "bundle exec yard config load_plugins true && bundle exec yard doctest"
 end
 
+desc "Run rubocop on yard examples."
+task :rubocop_yard_examples do
```

Comment:
@dazuma Can you think of a better name? Should this task be scoped under the `rubocop` namespace?